### PR TITLE
 minor edits and signup routes

### PR DESCRIPTION
**Pull request by PaulStorberg**

### CODE REVIEW

File: app/assets/stylesheets/shared/form.scss [hunk 1]
```diff
@@ -20,3 +23,28 @@
   margin-top: 20px;
   margin-bottom: 20px;
 }
+
+
+//////////////////////////////////////////////////////////////////////LOGIN FORM
+.background-container {
+  text-align: center;
```
Comment (kevinstorberg) [inline app/assets/stylesheets/shared/form.scss:30]: @PaulStorberg Alphabetize this

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -21,7 +21,7 @@ def edit
 
   def create
     @user.assign_attributes(user_params)
-    if @user.save
+    if @user.save!
```
Comment (kevinstorberg) [inline app/controllers/users_controller.rb]: @PaulStorberg Remove this bang

**Review by kevinstorberg — COMMENTED** [review]
Please make changes

File: app/assets/stylesheets/shared/form.scss (hunk 1 above)
Comment (PaulStorberg) [reply app/assets/stylesheets/shared/form.scss:30]: complete

File: app/controllers/users_controller.rb (hunk 2 above)
Comment (PaulStorberg) [reply app/controllers/users_controller.rb]: complete

**Pull request merged**